tensorboard/uploader/exporter_test.py

```diff
@@ -132,12 +134,24 @@ def stream_experiment_data(request, **kwargs):
         self.assertEqual(next(generator), "123")
         expected_files.append(os.path.join("experiment_123", "metadata.json"))
         expected_files.append(os.path.join("experiment_123", "scalars.json"))
+        expected_files.append(os.path.join("experiment_123", "tensors.json"))
         # blob_sequences.json should exist and be empty.
         expected_files.append(
             os.path.join("experiment_123", "blob_sequences.json")
         )
         self.assertCountEqual(expected_files, _outdir_files(outdir))
 
+        # Check that the tensors and blob_sequences data files are empty, because
+        # there are no tensors or blob sequences.
+        with open(
+            os.path.join(outdir, "experiment_123", "tensors.json")
+        ) as infile:
+            self.assertEqual(infile.read(), "")
+        with open(
+            os.path.join(outdir, "experiment_123", "blob_sequences.json")
+        ) as infile:
```

Comment:
Thankyou for adding check for blob_sequences file

Reply:
Ack.